.circleci/config.yml

```diff
@@ -82,18 +85,17 @@ jobs:
           condition:
             equal: [<< pipeline.git.branch >>, "main"]
           steps:
-            - overcommit
+            - quality
       - unless:
           condition:
             equal: [<< pipeline.git.branch >>, "main"]
           steps:
-            - run: echo "overcommit only runs on main branch"
+            - run: echo "quality only runs on main branch"
+            - quality # TODO
```

Comment:
[punchlist] Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker